WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+
+class CommentContentTableViewCell: UITableViewCell, NibReusable {
+
+    // determines the state of the like button.
+    enum LikeState {
+        case unliked
+        case liked
+    }
+
+    // all the available images to display for the accessory button.
+    enum AccessoryButtonType {
+        case share
+        case threeDots
+    }
+
+    // MARK: - Public Properties
+
+    var displayName: String? = "" {
```

Comment:
I'm assuming there will be a `configure` type method that will accept the `Comment`? That way all these `var`s can be set internally and thus can be private?

Reply:
Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/17109/commits/05d5a31e08a13244e2a493f9cf2aea0b40f07f88.